libs/libc/pthread/pthread_attr_setstack.c

```diff
@@ -39,9 +35,9 @@
  * Description:
```

Comment:
Function description still missing. Maybe we could use the documentation from the OpenGroup specification:
```suggestion
 * Description:
 *   The pthread_attr_setstack() function shall set the thread creation stack
 *   attributes stackaddr and stacksize in the attr object.
```

Reply:
Done.